### PR TITLE
[FTAMS102020] Nina van den Braak

### PR DESCRIPTION
**Pull request by ninsajala**
made a basic set-up and some of the styling, mainly need to finish last section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,14 +5,65 @@
...
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <main>
+      <nav>
+        <img src="images/spotify-logo.png" alt="spotify logo">
+        <div>
+        <a href="#">Premium</a>
```
Comment (wax-bear) [inline index.html:16]: The anchor elements could be indented.

File: index.html [hunk 2]
```diff
@@ -5,14 +5,65 @@
...
+        <a href="#">Discover</a>
+        <a href="#">Help</a>
+        <a href="#">Download</a>
+        </div>
+
```
Comment (wax-bear) [inline index.html:21]: Does this blank line serve a purpose?

File: styles/style.css [hunk 3]
```diff
@@ -1,8 +1,172 @@
 /*
 Colors:
 
-Text: 1A1A1A
-Green: #00B172
+Text: 
+Green: 
 White: #FFF
 
 */
```
Comment (wax-bear) [inline styles/style.css:8]: Is this comment of any use?

**Review by wax-bear — COMMENTED** [review]
Great stuff, Nina!

The website clone resonates well with the original.

Regarding the HTML/CSS a few comments:
- The naming of the classes and IDs can be improved. "#example" and ".explanation" work, and I can understand their meaning when seeing the website, but ideally they would be less generic and more specific in describing what they select. Of what is the section selected as "#example" an example? An artist? An artist profile? And what is explained in ".explanation"? These questions could clarify what would be more specific.
- Please mind the use of blank lines both in the HTML and CSS

File: styles/style.css (hunk 3 above)
Comment (ninsajala) [reply styles/style.css:8]: Nope, forgot to take it out, was already in there to give some hints about the coloring.
Thanks for the feedback, will implement it!

**Pull request closed without merging**